e2e/cypress/tests/integration/enterprise/elasticsearch_autocomplete/helpers/index.ts

```diff
@@ -0,0 +1,260 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
Even though I renamed the file, git considers it as a new file for some reason. If you have any suggestions to fix this, kindly let me know